control/rpmmacros.in

```diff
@@ -1,4 +1,3 @@
-%_topdir	%(echo $BUILDDIR)
```

Comment:
Why bother and change this line here if you remove the whole file in another commit?

Reply:
Your're right, but this is only because I've split my changes into several commits. The first one fixes the RPM build and contains this change. Then the second one is to simplify the RPM build process and removes this file.